physical/raft/fsm.go

```diff
@@ -672,7 +673,10 @@ func (f *FSM) ApplyBatch(logs []*raft.Log) []interface{} {
 							go f.restoreCb(context.Background())
 						}
 					default:
-						return fmt.Errorf("%q is not a supported transaction operation", op.OpType)
+						if _, ok := f.unknownOpTypes.Load(op.OpType); !ok {
```

Comment:
I'm ok with this, but I'll note that I don't think a sync.Map or any locking is necessary.  Even if raft-lib were to invoke ApplyBatch multiple times concurrently (which I don't think would make sense), boltdb Update transactions can't run concurrently with one another.

Reply:
Ah yes, of course. I noticed that the fsm itself grabbed a read lock before running the update transaction, so that's what got me thinking that some kind of race protection was necessary.